includes/definitions/eaton-sc200.yaml

```diff
@@ -0,0 +1,33 @@
+os: eaton-sc200
+text: 'Eaton SC200 Controller'
+type: power
+icon: eaton
+snmp_bulk: false
+over:
+    - { graph: device_current, text: Load }
+    - { graph: device_voltage, text: Voltage }
+mib_dir: eaton
+discovery:
+    -
+        sysObjectID: .1.3.6.1.4.1.1918.2.13
+discovery_modules:
+    mempools: false
+    cisco-vrf-lite: false
+    ospf: false
+    ports-stack: false
+    entity-physical: false
+    processors: false
+    storage: false
+    hr-device: false
+    discovery-protocols: false
+    bgp-peers: false
+    vlans: false
+    ucd-diskio: false
+    stp: false
+    ntp: false
+    fdb-table: false
+
+poller_modules:
+    mempools: false
+    cisco-vrf-lite: false
+    ospf: false
```

Comment:
Please remove these, only add a explict disable if they are causing issues

Reply:
> Please remove these, only add a explict disable if they are causing issues

I specifically disabled these because the CPU is woefully underpowered on these controllers, so was aiming to minimise the number of SNMP requests sent to the box. If that's the wrong way to do it, I'm happy to remove this from the model.

Reply:
Okay, I've pushed another commit with those disables removed